all/build.gradle

```diff
@@ -50,6 +50,7 @@ javadoc {
 
 task jacocoMerge(type: JacocoMerge) {
     dependsOn(subprojects.jacocoTestReport.dependsOn)
+    dependsOn(project(':grpc-interop-testing').jacocoTestReport.dependsOn)
     mustRunAfter(subprojects.jacocoTestReport.mustRunAfter)
```

Comment:
Not sure if need `mustRunAfter(project(':grpc-interop-testing').jacocoTestReport.mustRunAfter)` as well.

Reply:
`dependsOn` seems to be more reasonable than `mustRunAfter`. In terms of ordering for the two tasks, they are the same, you don't need to have both. If A dependsOn B, then A mustRunAfter B (plus other stuffs useful for incremental build/tasks).

Reply:
mustRunAfter is for things like clean+build. It is important for clean to run before build, but build does not have a dependency on clean.

Reply:
Oh, you're right. I see what you're saying now. I do need to copy those through. Things like `test` may be a mustRunAfter for merge.